dts/arm/nxp/nxp_lpc55S6x_common.dtsi

```diff
@@ -192,6 +192,8 @@
 
 	hs_lspi: spi@9f000 {
 		compatible = "nxp,lpc-spi";
+		cs-gpios = <&gpio0 20 0>,<&gpio1 1 0>,<&gpio1 12 0>,
+			<&gpio1 26 0>;
```

Comment:
Why force the hs_lspi node to use GPIO-based chip selects? The built-in chip select lines should work fine?

Reply:
Board specific configuration should not be done in dtsi files which should remain soc specific and board agnostic.